### DISCUSSION TITLE
Problems with sprintDoc template - collapsible sidebar

### DISCUSSION THREAD

**Discussion by athanhat** [combostrap/combo#30] [General]
Hi I have DokuWiki (Release 2020-07-29 "Hogfather") and I installed your combo plugin. 
It's really a great plugin and kudos for sharing it with all of us but unfortunately I ran into several problems:

1. The logout icon disappeared from my site
2. Site tools and User tools are there but the links are not functioning any more
3. All icons and menu items that I had created in sidebar disappeared

The problem appears to be the DokuWiki template I use, `sprintDoc` by Jana Deutschlaender
To escape from this malfunctioning state I had to disable your plugin and refresh the web page. 

FYI: I did change the template to the default one `dokuwiki` and all problems I had disappeared.
I am not sure what functionality I am missing by loosing `sprintDoc`, collapsible sidebar is definitely one. If you have any ideas how to make a collapsible sidebar please let me know.

**Comment by athanhat** [comment]
OK after some reading of your excellent documentation I found a solution to my problem
1. I installed your [Strap template][2]
2. Dokuwiki sidebar is the side slot in ComboStrap, it is explained [here][2]

[1]: https://combostrap.com/page/strap-side-slot-nxhixvt6
[2]: https://combostrap.com/get-started/strap-template-installation-combostrap-getting-started-nuvx767h

**Comment by gerardnico** [comment]
Conflict with css are pretty hard to solve.

Do you still have the problem ?

May be I should ask to install strap first in the getting started guide ?

**Comment by athanhat** [comment]
It works fine with Strap template and the content of my sidebar has become the content of side_slot and the `Sidebar page name` value is changed in [dokuwiki configuration][1]

Also to save some time for those that have similar problems with the sidebar, I found out how to create the collapsible tree that you have in ComboStrap Docs and this is indeed very cool. Magic happens with [Page Explorer navigational component][2]

Finally there is one thing I have not found so far in the documentation which is how to make ComboStrap side_slot or any slot collapsible, that would be nice to know.

[1]: https://combostrap.com/app/step-by-step/how-to-set-a-configuration-value-mej724rl
[2]: https://combostrap.com/navigation/page-explorer/adding-page-navigation-easily-to-your-website-i6zpgpwn#default_tree

**Comment by gerardnico** [comment]
There is already the toggle element that can make element collapsible
https://combostrap.com/animation/collapse-or-expand-any-component-with-the-toggle-effect-vh2s4f10

The area in the layout does not have for the moment any id, I will add one in the next release.

One problem is that by default, the collapsible element are not visible. It requires CSS expertise.

**Comment by gerardnico** [comment]
One things that you can now do is to wrap the element that you want to collapse in a box:
https://combostrap.com/block/combostrap-ui-box-5ysv3xg4
